common/views.py

```diff
@@ -170,72 +169,48 @@ def get_user_profile_info(self, request):
 
     def get(self, request, *args, **kwargs):
         user_profile_data = self.get_user_profile_info(request)
+        common_user = True if not request.user.is_superuser else False
```

Comment:
```suggestion
        common_user = not request.user.is_superuser
```

Reply:
And superuser can be a regular promoted user, so this would be misleading.

What I suggest us doing is keeping it a broken page for now, but add a jira issue to populate default super user with this data. We should bring it to the broad group.